precompiles/bank/bank.go

```diff
@@ -159,9 +171,44 @@ func (p Precompile) send(ctx sdk.Context, method *abi.Method, args []interface{}
 	if err != nil {
 		return nil, err
 	}
+
 	if err := p.bankKeeper.SendCoins(ctx, senderSeiAddr, receiverSeiAddr, sdk.NewCoins(sdk.NewCoin(denom, sdk.NewIntFromBigInt(amount)))); err != nil {
 		return nil, err
 	}
+
+	return method.Outputs.Pack(true)
+}
+
+func (p Precompile) sendNative(ctx sdk.Context, method *abi.Method, args []interface{}, caller common.Address, value *big.Int) ([]byte, error) {
+	pcommon.AssertArgsLength(args, 1)
+	if value == nil || value.Sign() == 0 {
+		return nil, errors.New("set `value` field to non-zero to send")
+	}
+
+	senderSeiAddr, ok := p.evmKeeper.GetSeiAddress(ctx, caller)
+	if !ok {
+		return nil, errors.New("invalid addr")
+	}
+
+	receiverAddr, ok := (args[0]).(string)
+	if !ok || receiverAddr == "" {
+		return nil, errors.New("invalid addr")
+	}
+
+	receiverSeiAddr, err := sdk.AccAddressFromBech32(receiverAddr)
+	if err != nil {
+		return nil, err
+	}
+
+	usei, wei, err := pcommon.HandlePaymentUseiWei(ctx, p.evmKeeper.GetSeiAddressOrDefault(ctx, p.address), senderSeiAddr, value, p.bankKeeper)
```

Comment:
NOTE: We can also just not use this `HandlePaymentUseiWei` for `sendNative` and directly bank send from the precompileAddr to the receiver addr. Just left it in case we specifically for it to be debited from user's sei address